node/actors/consensus/src/leader/replica_prepare.rs

```diff
@@ -1,12 +1,44 @@
 use super::StateMachine;
-use crate::{inner::ConsensusInner, leader::error::Error, metrics};
+use crate::{inner::ConsensusInner, metrics};
 use concurrency::ctx;
 use network::io::{ConsensusInputMessage, Target};
 use rand::Rng;
 use roles::validator;
 use std::collections::HashMap;
 use tracing::instrument;
 
+#[derive(thiserror::Error, Debug)]
+#[allow(clippy::missing_docs_in_private_items)]
+pub(crate) enum Error {
+    #[error("past view/phase (current view: {current_view:?}, current phase: {current_phase:?})")]
+    Old {
+        current_view: validator::ViewNumber,
+        current_phase: validator::Phase,
+    },
+    #[error("we are not a leader for this message's view")]
+    WhenNotLeaderInView,
```

Comment:
Bikeshedding: IMO, `When` prefix is redundant; `NotLeaderInView` would look better.

Reply:
done